Sentinel/Classes/UserDefaults/UserDefaultsTool.swift

```diff
@@ -46,6 +46,8 @@ private extension UserDefaultsTool {
                 viewController.navigationController?.pushViewController(userDefaultsViewController, animated: true)
             }
         }
+        .sorted { $0.title < $1.title }
```

Comment:
Maybe we could think about adding a search bar as well? Not for this release, but somewhere in the future

Reply:
I do agree with that 👍 Feel free to add that to our board.